<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="PB1">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formWidget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QLabel" name="lbId">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbLastName">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbFirstName">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbMiddleName">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Middle name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbBirthDate">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Birth date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbPhone">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Phone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbYearOfZNO">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Faculty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbNumberOfCertificate">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Course</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbPinOfCertificate">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="leId">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leLastName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFirstName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leMiddleName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leBirthDate">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePhone">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leFaculty">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCourse">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leGroup">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>168</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="PB5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate_Obj">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreate_object"/>
    <addaction name="actionShow_object"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuCreate_Obj"/>
  </widget>
  <action name="actionCreate_object">
   <property name="text">
    <string>Create object</string>
   </property>
  </action>
  <action name="actionShow_object">
   <property name="text">
    <string>Show object</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
